### ISSUE TITLE
Scss_lint not working with Syntastic (on vim)

### ISSUE THREAD

**Issue by tolgaek** [causes/scss-lint#383]
I get this error (all other linters are working fine with syntastic)

`syntastic: error: checker scss/scss_lint returned abnormal status -1`

**Comment by sds** [comment]
Hey @tolgaek,

It would be helpful if you posted the SCSS code you were editing when Syntastic reported that error. Also, are you on the latest version of `scss-lint`?

**Comment by tolgaek** [comment]
Hi @sds,

Yeah I'm on version 0.34.0 (just confirmed) and this happens with any scss file (including an empty scss) file but in case you need to have a file. Here's the content of a file that throws the same message

```scss
.alert {
  background-color: $secondary-light;
}
````

**Comment by sds** [comment]
Ok, if it's happening for every file then the issue is on your end. Are you using a Ruby version manager like `rvm` or `rbenv`? If so, it's likely that if your currently active Ruby version does not have `scss-lint` installed that you will get an abnormal error code.

What happens when you close `vim` and run `scss-lint -v` outside of it?

FWIW, `scss-lint` does not ever return an exit status of `-1`.

**Comment by tolgaek** [comment]
Yes I do use rvm but I installed scss-lint on my current version. 

I can use scss-lint -v (which is how I checked the version) I can also lint files outside of vim.

Also `:SyntasticInfo` does show scss-lint as a valid linter (it doesn't show up there if not installed)

**Comment by sds** [comment]
What happens when you run `!scss-lint -v` from within `vim`?

Again, if that also works then I'm not sure how I can help here, as I can't reproduce this myself and `scss-lint` doesn't return `-1` as a status code, so it's likely an issue with an `rvm`/`rbenv` shim.

**Comment by tolgaek** [comment]
When running from within vim, I do get a stack trace with ruby related error. I included it below. I don't know what the issue is, but you can close this ticket if it's not related to scss-lint. I will try to figure it out, thanks
![trace](https://cloud.githubusercontent.com/assets/891261/6563198/9b1519dc-c675-11e4-8662-561894f996ba.png)

**Comment by tolgaek** [comment]
Figured it out! It was due to my zsh set up with rvm. Basically it's this weird thing with Vim not loading .zshrc file. More info is found here https://rvm.io/integration/vim

Basically you need to include `set shell=/bin/sh` in .vimrc to allow loading of rvm properly.

**Comment by sds** [comment]
I'm glad you figured out the problem, @tolgaek. Thanks for posting the solution! Hopefully this saves someone else some time.
